docs/introduction/Cross-chain-transfer-protocol.rst

```diff
@@ -209,6 +209,26 @@ Mainchain request can be performed through a raw transaction with the following
             "scProof": sc_proof1
         }]
 
+Circuit with key rotation
+===========================
+
+Circuit with key rotation is needed to replace compromised signers and masters keys of certificate submitters with new keys.
+These changes occur in-chain for 2 reasons:
+- every node must keep knowledge about the recent set of public keys. And if we keep this information off-chain we can easily loose it.
+- we need to be sure that all the nodes use exactly the same source of data for signing or verifying certificate, creating the snark proof, etc.
+Every key rotation transaction is validated according to a set of rules, then all key rotations within certificate submission epoch are aggregated, included to certificate, submitted to Mainchain. Starting from the next epoch previous keys are invalidated, and new keys are activated.
+The route for POST request is http://127.0.0.1:9085/transaction/createKeyRotationTransaction, caller should be authenticated to use it.
+Parameters for request are following:
+- 'keyType' of type Integer;
```

Comment:
Lets put indentation at the beginning on every line and make bold name of each parameter(** parameter name**).